### PR TITLE
Docs: Convert "reporting security issues" from RST to MD

### PR DESCRIPTION
**Pull request by tbrlpld**
Convert docs page "Reporting security issues" from Restructured Text to Markdown. 

@thibaudcolas Here is the first PR, as discussed yesterday. Let me know if something needs changing.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -21,7 +21,7 @@ ignore-path = _build,docs/_build
 [isort]
 line_length=100
 multi_line_output=4
-skip=migrations,project_template,node_modules,.git,__pycache__,LC_MESSAGES
+skip=migrations,project_template,node_modules,.git,__pycache__,LC_MESSAGES,venv
```
Comment (thibaudcolas) [inline setup.cfg:24]: Hmm, I can see why we might want to do this but it’s unrelated to the rest of the PR so would be better separately.

```suggestion
skip=migrations,project_template,node_modules,.git,__pycache__,LC_MESSAGES
```

File: docs/contributing/security.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+Once you've submitted an issue via email, you should receive an acknowledgement from a member of the security team within 48 hours, and depending on the action to be taken, you may receive further followup emails.
+
+If you want to send an encrypted email (optional), the public key ID for <security@wagtail.io> is `0x6ba1e1a86e0f8ce8`, and this public key is available from most commonly-used keyservers.
+
+Django security issues should be reported directly to the Django Project, following [Django\'s security policies](https://docs.djangoproject.com/en/dev/internals/security/) (upon which Wagtail\'s own policies are based).
+
+## Supported versions
+
+At any given time, the Wagtail team provides official security supportfor several versions of Wagtail:
```
Comment (thibaudcolas) [inline docs/contributing/security.md:21]:
```suggestion
At any given time, the Wagtail team provides official security support for several versions of Wagtail:
```

**Review by thibaudcolas — APPROVED** [review]
Thank you @tbrlpld! I’ll merge your two changes separately now.

File: setup.cfg (hunk 1 above)
Comment (tbrlpld) [reply setup.cfg]: I have reverted that in 4247756

**Comment by thibaudcolas** [comment]
@tbrlpld congratulations for your first Wagtail contribution!

Bilbo runs through the shire: I’m going on an adventure!

**Comment by tbrlpld** [comment]
Thanks for reviewing @thibaudcolas 😄

**Pull request merged**